Neos.ContentGraph.PostgreSQLAdapter/src/Domain/Repository/NodeFactory.php

```diff
@@ -92,6 +93,8 @@ public function mapNodeRowToNode(
                 $this->propertyConverter
             ),
             $nodeRow['nodename'] ? NodeName::fromString($nodeRow['nodename']) : null,
+            // TODO implement {@see \Neos\ContentGraph\DoctrineDbalAdapter\Domain\Repository\NodeFactory::mapNodeRowToNode()}
+            SubtreeTagsWithInherited::createEmpty(),
```

Comment:
I didn't work on the PostgreSQL adapter – that should be done post merge to keep this PR manageable IMO